upgrade_test.py

```diff
@@ -453,12 +453,15 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
         entire_write_cs_thread_pool = self.run_stress_thread(stress_cmd=write_stress_during_entire_test)
 
         # Let to write_stress_during_entire_test complete the schema changes
-        time.sleep(300)
+        self.metric_has_data(
+            metric_query='collectd_cassandra_stress_write_gauge{type="ops", keyspace="keyspace_entire_test"}', n=10)
 
         # Prepare keyspace and tables for truncate test
         if self.truncate_entries_flag:
             self.insert_rows = 10
             self.fill_db_data_for_truncate_test(insert_rows=self.insert_rows)
+            # Let to ks_truncate complete the schema changes
+            time.sleep(120)
```

Comment:
@roydahan FYI, some of the issue was cause of this code is doing schema changes, and we start `cassandra-stress` straight away